src/qlik-multi-kpi.less

```diff
@@ -90,10 +90,15 @@
     margin: 0;
     height: auto;
     align-self: flex-start;
+    flex-wrap: wrap;
```

Comment:
they should not wrap. For a horizontal layout they should just continue horizontally, adding a scroll when needed,

Reply:
fixed